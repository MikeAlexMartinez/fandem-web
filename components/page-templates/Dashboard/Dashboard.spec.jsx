import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import DashboardComponent from './Dashboard';

describe('<Dashboard />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<DashboardComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
