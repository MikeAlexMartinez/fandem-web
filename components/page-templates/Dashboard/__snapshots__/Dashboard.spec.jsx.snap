// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> renders and matches the snapshot 1`] = `
<Dashboard
  classes={Object {}}
/>
`;
